backend/app/usecase/url/urlretriever.go

```diff
@@ -13,12 +13,14 @@ var _ Retriever = (*RetrieverPersist)(nil)
 // Retriever represents URL retriever
 type Retriever interface {
 	GetURL(alias string, expiringAt *time.Time) (entity.URL, error)
```

Comment:
Since we are here, maybe changing the expiringAt type to time.Time would be good

Reply:
@arberiii This is part of the use case. API caller may not specify `expiringAt`.